tests/test.py

```diff
@@ -11,6 +11,20 @@ class Test:
     """
     A class for writing and running tests with a fluent API.
```

Comment:
What do you mean by "fluent" API?  You mean "easy to use"?

Reply:
Yes. I meant "easy to use".

Reply:
Updated in https://github.com/anand-kamble/data-filtering/pull/9/commits/d639c735d3a60db7d51c2bfc9c9cc1b74622c48c